.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Not sure what's happening here. Please feel free to add `.pytest_cache/` to `.gitignore`.